pkg/kp/rcstore/consul_store.go

```diff
@@ -220,7 +299,7 @@ func (s *consulStore) Watch(rc *fields.RC, quit <-chan struct{}) (<-chan struct{
 			case <-quit:
 				return
 			case <-time.After(1 * time.Second):
-				pairs, meta, err := s.kv.List(kp.RCPath(rc.Id.String()), &api.QueryOptions{
+				pairs, meta, err := s.kv.List(kp.RCPath(rc.ID.String()), &api.QueryOptions{
```

Comment:
Now that all fields of an RC reside under a single key, is it possible we can use `Get` rather than `List`?